e-commerce/src/components/asideCatalogCategory/asideCatalogCategory.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export type OnClickType = {
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+};
```

Comment:
Этот объект не нужен, можно описать тип самой функции
```suggestion
export type OnClickType = {
  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
};
```